gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/snapshots/transform/FindPreviewsAcrossProjects.kt

```diff
@@ -35,6 +35,8 @@ abstract class FindPreviewsAcrossProjects : DefaultTask() {
     // Clear any existing content
     output.writeText("")
 
+    logger.lifecycle("includePrivatePreviews = ${includePrivatePreviews.orNull}")
```

Comment:
If we only need this for debugging if so I think we should move to a lower log level like `info`